src/opnsense/mvc/app/models/OPNsense/Routes/Route/Migrations/M0_0_1.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace OPNsense\Routes\Route\Migrations;
+use OPNsense\Base\BaseModelMigration;
+class M0_0_1 extends BaseModelMigration
+{
+    public function run($model)
+    {
+       $model.save();
```

Comment:
@fabianfrz ok, sounds good, I will try to read the code tomorrow to see if there's work to be done in the current configd model class.